### PR TITLE
Dts title fixes

### PR DESCRIPTION
**Pull request by galak**

### CODE REVIEW

**Review by chrta — APPROVED** [review]

File: dts/bindings/sensor/aosong,dht.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2019 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-title: Aosong DHT Digital-output Humidity and Temperature Sensor
-
 description: |
+    Aosong DHT Digital-output Humidity and Temperature Sensor
```
Comment (ulfalizer) [inline dts/bindings/sensor/aosong,dht.yaml]: Could maybe make it

    Aosong DHT (Digital-output Humidity and Temperature sensor)

to make it clear that it's explaining the acronym.

File: dts/bindings/gpio/ti,boosterpack-header.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2019 Brett Witherspoon
 # SPDX-License-Identifier: Apache-2.0
 
-title: TI BoosterPack GPIO header
-
 description: |
+    TI BoosterPack GPIO header
```
Comment (ulfalizer) [inline dts/bindings/gpio/ti,boosterpack-header.yaml]: Maybe this line is redundant with the line after it ("GPIO pins exposed..."), or the information could be merged.

**Review by ulfalizer — APPROVED** [review]

**Review by nashif — APPROVED** [review]

File: dts/bindings/sensor/aosong,dht.yaml (hunk 1 above)
Comment (galak) [reply dts/bindings/sensor/aosong,dht.yaml]: Done

Comment (ulfalizer) [reply dts/bindings/sensor/aosong,dht.yaml]: Could make 's' lowercase since "sensor" isn't part of the acronym.

**Pull request merged**